internal/eigenState/avsOperators/avsOperators.go

```diff
@@ -194,12 +194,18 @@ func (a *AvsOperatorsModel) IsInterestingLog(log *storage.TransactionLog) bool {
 	return a.BaseEigenState.IsInterestingLog(addresses, log)
 }
 
-func (a *AvsOperatorsModel) InitBlockProcessing(blockNumber uint64) error {
+func (a *AvsOperatorsModel) InitBlock(blockNumber uint64) error {
 	a.stateAccumulator[blockNumber] = make(map[types.SlotID]*AccumulatedStateChange)
 	a.deltaAccumulator[blockNumber] = make([]*AvsOperatorStateChange, 0)
 	return nil
 }
 
+func (a *AvsOperatorsModel) CleanupBlock(blockNumber uint64) error {
```

Comment:
Rather than `InitBlock` and `ClearBlock` we should go with something like `SetupStateForBlock` and `CleanupProcessedStateForBlock`.

Reply:
https://github.com/Layr-Labs/sidecar/pull/62/commits/1e801ba5d5aeeabac725c43e01ac8f703b1e5265

prefer the clean names, but ok :)